ods_ci/tests/Tests/100__deploy/130__operators/140__serverless_operator/140__serverless_operator.robot

```diff
@@ -0,0 +1,111 @@
+*** Settings ***
+Documentation    Test Cases to verify Serverless installation
+Library         Collections
+Library         SeleniumLibrary
+Library         OpenShiftLibrary
+Resource        ../../../../Resources/Page/OCPDashboard/OCPDashboard.resource
+Suite Setup      Suite Setup
+Suite Teardown   Suite Teardown
+
+
+*** Variables ***
+${RHOAI_APPNAME}  Red Hat OpenShift AI
+${RHOAI_OPERATOR_NAME}    Red Hat OpenShift AI
+${KSERVE_APPNAME}  Red Hat OpenShift Serverless
+${KSERVE_OPERATOR_NAME}    Red Hat OpenShift Serverless
+${KNATIVESERVING_NS}    knative-serving
+${RHOAI_OPERATOR_NS}    redhat-ods-operator
+
+
+*** Test Cases ***
+Validate DSC creates all Serverless CRs
+    [Documentation]  The purpose of this Test Case is to validate the creation
+    ...    of KServe Custom Resources
+    [Tags]  Operator    ODS-2600
+    Check And Install Operator In Openshift    ${RHOAI_APPNAME}    ${RHOAI_OPERATOR_NAME}
+    Check And Install Operator In Openshift    ${KSERVE_APPNAME}    ${KSERVE_OPERATOR_NAME}
+    Is Resource Present     KnativeServing    knative-serving     ${KNATIVESERVING_NS}
+    Check Status     oc get KnativeServing knative-serving -n ${KNATIVESERVING_NS} -o json | jq '.status.conditions[] | select(.type=="Ready") | .status'     KnativeServing    "True"    # robocop: disable
+    Is Resource Present     Gateway    knative-ingress-gateway     ${KNATIVESERVING_NS}
+    Is Resource Present     Gateway    knative-local-gateway     ${KNATIVESERVING_NS}
+    Is Resource Present     Service    knative-local-gateway     istio-system
```

Comment:
why not set variable for istio-system just like others?

Reply:
because it is only used once @zdtsw

Reply:
Service Mesh CR namespace is specified in the DSCI, and thus can be other then 'istio-system'.

Reply:
@mattmahoneyrh This Test Case does not touch Service Mesh

Reply:
Let's use variable also for this one to be consistent, as it may be easier to update later on.

Also, in case it's specified in DSCi, maybe we can read it's value from there? Otherwise we shall have different values prepared per the product type, which can be determined based on `PRODUCT` variable in `tests-variables.yml`.

Reply:
Added